assets/js/components/ReportZero.stories.js

```diff
@@ -32,23 +29,28 @@ export function ReportZeroStory() {
 }
 ReportZeroStory.storyName = 'Report Zero';
 ReportZeroStory.decorators = [
-	( Story ) => {
-		const registry = createTestRegistry();
-		const testModuleDefinition = createModuleStore( 'test-module', {
-			storeName: 'modules/test-module',
-		} );
-		registry.registerStore(
-			testModuleDefinition.STORE_NAME,
-			testModuleDefinition
-		);
-		provideModules( registry, [
-			{ slug: 'test-module', name: 'Test Module' },
-		] );
+	( Story, { parameters } ) => {
+		const setupRegistry = ( registry ) => {
+			const testModuleDefinition = createModuleStore( 'test-module', {
+				storeName: 'modules/test-module',
+			} );
+			registry.registerStore(
+				testModuleDefinition.STORE_NAME,
+				testModuleDefinition
+			);
+			provideModules( registry, [
+				{ slug: 'test-module', name: 'Test Module' },
+			] );
+
+			if ( parameters?.setupRegistry ) {
+				parameters.setupRegistry( registry );
+			}
+		};
 
 		return (
-			<WithTestRegistry registry={ registry }>
-				<Story />
-			</WithTestRegistry>
+			<WithRegistrySetup func={ setupRegistry }>
+				<Story { ...parameters } />
```

Comment:
Again, this spreading into the story here shouldn't be needed, but even if it was, this should be `args` not `parameters`.